client/components/plans/plan-price/index.jsx

```diff
@@ -23,8 +23,9 @@ const PlanPrice = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
-			const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
-			formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
+			const currencySymbol = formattedPrice.slice( 0, 1 );
```

Comment:
Good point, though there isn't a clean way to fix this until we pass down the current currency code. Current currencies we support are USD, AUD, CAD, EUR, GBP and JPY`